### PR TITLE
Fix movement when far away from the center of the world

### PR DESCRIPTION
**Pull request by AJ-Ferguson**

### CODE REVIEW

**Comment by rtm516** [comment]
Should close https://github.com/GeyserMC/Geyser/issues/204 needs testing

File: connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockMovePlayerTranslator.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public void translate(MovePlayerPacket packet, GeyserSession session) {
         double javaY = packet.getPosition().getY() - EntityType.PLAYER.getOffset();
         if (packet.isOnGround()) javaY = Math.ceil(javaY * 2) / 2;
         ClientPlayerPositionRotationPacket playerPositionRotationPacket = new ClientPlayerPositionRotationPacket(
-                packet.isOnGround(), GenericMath.round(packet.getPosition().getX(), 4), javaY, GenericMath.round(packet.getPosition().getZ(), 4), packet.getRotation().getY(), packet.getRotation().getX()
+                packet.isOnGround(), Double.parseDouble(Float.toString(packet.getPosition().getX())), javaY, Double.parseDouble(Float.toString(packet.getPosition().getZ())), packet.getRotation().getY(), packet.getRotation().getX()
```
Comment (DoctorMacc) [inline connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockMovePlayerTranslator.java:71]: Can you add a comment explaining why we're converting from a float to string to double?

**Comment by AnonAlpha** [comment]
> Should close #204 needs testing
> 
> Edit: Confirmed to fix #204

Confirmed to fix #204 on plain ground, still can't fix stairs

File: connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockMovePlayerTranslator.java (hunk 1 above)
Comment (RednedEpic) [reply connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockMovePlayerTranslator.java:73]: Done.

**Pull request merged**